packages/mobile/components/TopBar.svelte

```diff
@@ -0,0 +1,13 @@
+<script>
+    import { AccountSwitcher, AccountActionsButton } from './'
+
+    export let profileInitial = 'A'
+</script>
+
+<div class="flex flex-row justify-between">
+    <button class="w-10 h-10 relative flex items-center justify-center rounded-full bg-blue-500 leading-100">
```

Comment:
We should add all these features to the feature flags so they can be enabled and disabled, and they disappear if they are not enabled. imo we need it for 
- profile button
- account siwtcher
- account actions button